### PR TITLE
IfSG improvements

### PR DESCRIPTION
**Pull request by xuwhite**
This PR adds an admin page which allows supporters, shicos and team coordinators to confirm and change a users ifsg certificate.

### CODE REVIEW

File: src/Controllers/SettingsController.php [hunk 1]
```diff
@@ -263,48 +260,42 @@ public function certificate(): Response
 
     public function saveIfsgCertificate(Request $request): Response
     {
-        if (!config('ifsg_enabled')) {
+        $user = $this->auth->user();
+        $canConfirm = $this->auth->can('user.ifsg.edit');
+        if (!config('ifsg_enabled') || $user->license->ifsg_confirmed && !$canConfirm) {
```
Comment (xuwhite) [inline src/Controllers/SettingsController.php:268]: Throw not found if ifsg_confirmed and !canConfirm? really?

File: src/Controllers/SettingsController.php [hunk 2]
```diff
@@ -263,48 +260,42 @@ public function certificate(): Response
...
-                    $ifsg_confirmation_change = $user->license->ifsg_confirmed !== (bool) $data['ifsg_confirmed'];
-                    $user->license->ifsg_confirmed = (bool) $data['ifsg_confirmed'];
-                }
-            }
-            $user->license->save();
-
-            $this->addNotification('settings.certificates.success');
-
-            $ifsg_confirmation_change
-                ? $this->log->info('Certificate {certificate} of user {user} ({id}) was {confirmed}.',
```
Comment (xuwhite) [inline src/Controllers/SettingsController.php]: no log message any more?

File: resources/lang/en_US/default.po [hunk 3]
```diff
@@ -449,7 +449,7 @@ msgid "settings.certificates.ifsg_confirmed.hint"
 msgstr "Your health instruction has been confirmed, so you can no longer change your data."
 
 msgid "settings.certificates.confirmed.info"
-msgstr "You personally checked the certificate / license"
+msgstr "You personally checked the certificate / license and it mets the requirements"
```
Comment (xuwhite) [inline resources/lang/en_US/default.po:452]: Do you mean "meets the requirements"?

Comment (MyIgel) [reply resources/lang/en_US/default.po]: I mean "coding while half asleep might sometimes lead to typing mistakes" :upside_down_face:

**Pull request merged**